app/views/users/index.html.erb

```diff
@@ -1 +1,9 @@
-<h1>user index</h1>
\ No newline at end of file
+<% if @users.empty? %>
+  No users
+<% else %>
+  <% @users.each do |user| %>
+    <%= link_to user_path(user) do %>
+      <%= render user %>
+    <% end %><br>
+  <% end %>
+<% end %>
```

Comment:
- [ ] @geekelo I am getting the error below  when I try to access the users' page, to fix it you can delete the 4th line code  from your manifest.js file, you currently don't have a directory called vendors in the assets folder that's why the error is showing
![image](https://github.com/geekelo/rails-blog_app/assets/61408860/03f345a3-1dfa-42a3-a126-47d3a0574f7e)

![image](https://github.com/geekelo/rails-blog_app/assets/61408860/43ccb5e5-c0be-4728-a224-9e97e838250c)